composer.json

```diff
@@ -24,28 +24,25 @@
         }
     ],
     "require": {
-        "php": "^5.5.9|^7.0",
-        "symfony/framework-bundle": "^2.7.22|~3.0|~4.0",
-        "symfony/console": "~2.7|~3.0|~4.0",
-        "symfony/dependency-injection": "~2.7|~3.0|~4.0",
-        "doctrine/dbal": "^2.5.12",
-        "jdorn/sql-formatter": "^1.2.16",
-        "symfony/doctrine-bridge": "~2.7|~3.0|~4.0",
-        "doctrine/doctrine-cache-bundle": "~1.2"
+        "php": "^7.2",
```

Comment:
Since SF 4 requires PHP 7.1, this bundle should IMO do the same, at least until 5.0 GA.

Reply:
Yes, this needs to stay 7.1 for now.

Reply:
No need to follow symfony's dependency versioning: the bundle can define its own dependency versions (and it should)

Reply:
This is a bundle for the Symfony eco-system and it is built for Symfony and should follow the Symfony release cycle they follow and document publicly.

Reply:
This is something we discussed, discussed and discussed again ad-nauseam.

If the package still has to live under `doctrine/*`, then keep it under our maintenance with our constraints and our ability to maintain it.

If it is part of the Symfony LTS cycle yaddayaddawhatever, then let's push it under `symfony/*` and call it a day.

Reply:
Also (and extremely important), this targets `master`, not the current release. Dependency bumps are indeed for the next minor version.

Reply:
I think it is best if we follow the release roadmap that Symfony publicly documents.

If you merge this to master and release it as 1.10, then when Symfony releases a 4.x version with new features and we add them in 1.11, DoctrineBundle will be incompatible with Symfony and the 7.1 users won't be able to upgrade Symfony.

DoctrineBundle is a part of the Symfony eco-system and we have people from the Symfony team (@weaverryan @nicolas-grekas @javiereguiluz) that actively help maintain this bundle so I don't think it is a problem for us to play nice with the Symfony eco-system.

Reply:
> and the 7.1 users won't be able to upgrade Symfony.

This is not a problem

Reply:
Symfony eco-system still supports 7.1 and this bundle is built for that eco-system.

Reply:
Again: not a problem. You get the new features when you get to upgrade.

As mentioned above, this is a bundle, and if it should be maintained outside the doctrine organisation then I don't understand why it is in here in first place. If the release goes hand in hand with symfony, make it part of the symfony subtree split..

Reply:
Because that is just where it ended up several years ago. Where it is in github is irrelevant imo. Github has features to handle this, we have outside collaborators who have the ability to maintain this bundle (and they do). You can control notifications per repository, etc.

I can't think of any reason why it can't be moved but I also don't think it is a very big deal for the repository to live here and follow the documented release lifecycle of Symfony.

Reply:
>  I also don't think it is a very big deal for the repository to live here and follow the documented release lifecycle of Symfony.

It is a big deal for my inbox, from what I can see.

Alright then, closing as `won't fix` here, will simply mute the repo.